includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -2107,12 +2107,6 @@ static function() {
 					.primary-navigation > div > .menu-wrapper > .menu-item-has-children:hover > .sub-menu-toggle > .icon-minus {
 						display: flex;
 					}
-
-					/* In the Twenty Twenty-One theme, when on a non-AMP page and the mobile menu is open, the switcher
-					is incorrectly shown at the top. In that case the switcher is hidden. */
-					body.lock-scrolling > #amp-mobile-version-switcher {
-						display: none;
-					}
```

Comment:
@pierlon I removed this in a70cf572c91d41d9a7e9286746fdd874ebd0c073 because the CSS you added to `MobileRedirection` is printed on AMP and non-AMP pages alike.